src/redux/root.saga.js

```diff
@@ -14,6 +14,7 @@ import checkoutSaga from './checkout/checkout.sagas';
 import chatSaga from './chat/chat.sagas';
 import headerLinksSaga from './header-links/header-links.sagas';
 import homeLooksImagesSaga from './home-page-looks/home-page-looks.sagas';
+import HomePageSliderSaga from './homepage-slider/homepage-slider.sagas';
```

Comment:
Please, lower case that import to according of rest above.

Reply:
fixed